samigo/samigo-app/src/webapp/jsf/author/item/fillInNumeric.jsp

```diff
@@ -35,7 +35,7 @@
       <title><h:outputText value="#{authorMessages.item_display_author}"/></title>
       <samigo:script path="/js/authoring.js"/>
       </head>
-      <body onload="countNum();<%= request.getAttribute("html.body.onload") %>">
+      <body onload="countNum();<%= request.getAttribute("html.body.onload") %>">

```

Comment:
I couldn't get fillInNumeric.jsp formatted in a way that it wouldn't commit these non-changes. Even VIM thinks the two lines are identical. Could it be something with the line endings? I'm developing on linux.